euclid.json

```diff
@@ -39,9 +39,21 @@
       "monitoring"
     ]
   },
-  "ansible": {
-    "hosts_file": "infra/ansible/hosts.ansible.yml",
-    "configure_playbook_file": "infra/ansible/playbooks/configure.ansible.yml",
-    "deploy_playbook_file": "infra/ansible/playbooks/deploy.ansible.yml"
+  "deploy": {
+    "network": {
+      "name": "testnet|integrationnet|mainnet",
+      "node": {
+        "ip": ":gl0_node_ip",
+        "id": ":gl0_node_id",
+        "public_port": ":gl0_node_public_port"
+      }
+    },
+    "ansible": {
+      "hosts": "infra/ansible/hosts.ansible.yml",
+      "playbooks": {
+        "deploy": "infra/ansible/playbooks/deploy/deploy.ansible.yml",
+        "start": "infra/ansible/playbooks/start/start.ansible.yml"
+      }
+    }
```

Comment:
I feel like this info should be under the "deploy" key rather than next to it.

Reply:
But it's already under `deploy` key
``` 
  "deploy": {
    "network": {
      "name": "testnet|integrationnet|mainnet",
      "node": {
        "ip": ":gl0_node_ip",
        "id": ":gl0_node_id",
        "public_port": ":gl0_node_public_port"
      }
    },
    "ansible": {
      "hosts": "infra/ansible/hosts.ansible.yml",
      "playbooks": {
        "deploy": "infra/ansible/playbooks/deploy/deploy.ansible.yml",
        "start": "infra/ansible/playbooks/start/start.ansible.yml"
      }
    }
  }
```

Reply:
Oh whoops, I must be reading too fast. Great - I think it makes sense under "deploy" as a sibling of "network" like you have it.